Source/WebCore/PAL/pal/graphics/WebGPU/WebGPUCompositorIntegration.h

```diff
@@ -44,6 +44,7 @@ class CompositorIntegration : public RefCounted<CompositorIntegration> {
 
 #if PLATFORM(COCOA)
     virtual Vector<MachSendRight> recreateRenderBuffers(int width, int height) = 0;
+    virtual void setWebProcessID(unsigned) { }
```

Comment:
Right, this is a perfect example of why the Compositor Integration object exists - it's our own object with our own API that we fully control.